src/marqo/models/search_models.py

```diff
@@ -26,3 +30,28 @@ class BulkSearchBody(SearchBody):
 class BulkSearchQuery(StrictBaseModel):
     queries: List[BulkSearchBody]
 
+
+class RetrievalMethod(str, Enum):
+    Disjunction = 'disjunction'
+    Tensor = 'tensor'
+    Lexical = 'lexical'
+
+class RankingMethod(str, Enum):
+    RRF = 'rrf'
+    NormalizeLinear = 'normalize_linear'
+    Tensor = 'tensor'
+    Lexical = 'lexical'
+
+class HybridParameters:
```

Comment:
Should this be a pydantic class and the field names are in `camelCase`?

Reply:
Ah yep it should be a pydantic class. We want the field names in snake case though, right?

Reply:
Yes, the original field name should be in the snake case for the Python client user. Note that the query sent to Marqo should be converted to the `camelCase` as APIs using the alias feature.